### ISSUE TITLE
Feature Request: Unlocked Cutscenes

### ISSUE THREAD

**Issue by Relo32** [EnderHDMC/MHWISaveEditor#35]
Hello, I'm interested in knowing if it's possible to mark all the cutscenes in the game as unlocked since this could solve the problem that new players have with the story mode in multiplayer.

**Comment by EnderHDMC** [comment]
This should be quite simple to do, the cutscene unlocks are already known, it would just be a matter of adding it to the app.

**Comment by Relo32** [comment]
That's great, thanks for looking into it.

**Comment by KodaichiZero** [comment]
Even though this is an old request, I'd like to further this. I really want to see if this improves the multiplayer experience for my friend and I. But if the cutscene unlocks are "known", please help me find where. I'll dive in there myself with a hex editor if I have to.

**Comment by Relo32** [comment]
Yeah, i still believe it's a very important feature for lots of players even today, if it would solve the one issue with multiplayer people had since the game came out

**Comment by EnderHDMC** [comment]
> Even though this is an old request, I'd like to further this. I really want to see if this improves the multiplayer experience for my friend and I. But if the cutscene unlocks are "known", please help me find where. I'll dive in there myself with a hex editor if I have to.

The data is mapped in 010 Editor:
```
You'll need 010 Editor.

In my save editor, just dump your save file in `File -> Dump -> SAVEDATA1000.bin`.
That'll save an unencrypted version of your save file in the same place as your save.
You can go `File -> Open save location` to get there.

Open that file in 010 Editor, then go to `Templates -> Open Template`, and navigate to my save editor.
In there go to `res/mapping/010 Editor/` and open a file called `SAVEDATA1000.bt`.

Next, in the `Template Results`:
Go to `mhw_save -> section3 -> saves[your slot]`, if you've selected the right slot, you should see your character's name.
In there look for `gallery_unlocked` and `gallery_unseen`, it'll be just above `unknown21a`.

Those should have the values:
gallery_unlocked = FE FF FF FF DD E6 BF FF DD FF FF FF FF FF 7F 00
gallery_unseen   = 00 00 00 00 00 00 00 00 01 00 00 00 00 00 00 00
```

**Comment by KodaichiZero** [comment]
I finally got the chance to test this out. This fix does in fact unlock all of the game's cutscenes, verifiable by checking the gallery on the title screen. Also seems to let players join a story assignment, but only after the host of the assignment has seen the cutscene. So you can effectively play through the game's story with your friend, but once again, usually you won't actually be able to join the mission until the host has encountered the monster. That seems to be a completely separate requirement from having seen the cutscene previously.

Basically, the solution above improves things a bit but is not a perfect solution for players wanting to play through the game's story together. Another mod would likely have to be made to disable that hard requirement from assignments.

Still though, thank you so much for coming through with this. My friend and I have decided this is a fair compromise, compared to not being able to do assignments together at all.
